api/room/schema.py

```diff
@@ -226,6 +244,25 @@ def resolve_room_schedule(self, info, calendar_id, days):
             events=room_schedule[1]
         )
 
+    @Auth.user_roles('Admin')
+    def resolve_analytics_for_room_least_used_per_week(self, info, location_id, week_start, week_end):  # noqa: E501
+        query = Room.get_query(info)
+        room_analytics = RoomAnalytics.get_least_used_room_week(
+            self, query, location_id, week_start, week_end
+        )
+        return Analytics(
+            analytics=room_analytics
+        )
+
+    def resolve_analytics_for_meetings_per_room(self, info, location_id, day_start, day_end):  # noqa: E501
```

Comment:
@kevinbett I suggest you restrict this query to admins only as specified in the task description otherwise good work.